vignettes/NSSP_nowcast.Rmd

```diff
@@ -0,0 +1,539 @@
+pkgdown:
+  as_is: true
+bibliography: library.bib
+csl: https://raw.githubusercontent.com/citation-style-language/styles/master/apa-numeric-superscript-brackets.csl
+link-citations: true
+vignette: >
+  %\VignetteIndexEntry{Nowcasting syndromic surveillance system data: a case study applied to the U.S. National Syndromic Surveillance Program (NSSP) data}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+# Introduction
+
+Syndromic surveillance data are used by public health departments to understand trends in clinical encounters from electronic health records.
+When these datasets are analysed in real-time, delays from the initial visit to updated diagnoses codes results in right-truncation of counts of the primary event, i.e. the patient visit, leading to systematic downward bias in the most recent data.
+Nowcasting uses the previously observed reporting delays to correct for this downward bias and estimate the eventual, final observed cases, which provides an estimate of the trend in cases in real-time.
+
+## About syndromic surveillance system data
+
+Syndromic surveillance system data contains information at the visit-level on the timing and nature of the patient's clinical interactions.
+In this case study, we will use synthetic data from the United States' [National Syndromic Surveillance Program](https://www.cdc.gov/nssp/index.html) (NSSP) dataset available for participating jurisdictions through the CDC's ESSENCE platform, to nowcast cases of Broad Acute Respiratory Incidence (BAR).
+The NSSP Emergency Department (ED) visit dataset is a dataset widely used by public health departments in the United States, representing many but not all counties in the country.
+The dataset contains information at the visit-level about each clinical encounter recorded in the electronic health record system during the patient's hospital stay.
+Clinical encounters may begin to be associated with diagnoses codes at different points in the patient or clinical processing journey (e.g. during registration, triage, clinical encounter, after laboratory results are returned, or during coding for billing, etc.), and are captured through update messages to the syndromic surveillance system once they are entered into a facility's electronic health record.
+The difference between the visit date - when the patient registers in the emergency department- and the time of the diagnosis update pertaining to the diagnosis of interest, is used to compute a reporting delay for each patient.
+Reporting delays for diagnoses of interest can vary by a range of factors including by pathogen/syndrome, season, time of day or week, means of diagnosis, the electronic health record system, or treating facility.
+In this vignette, we will demonstrate how to use syndromic surveillance system data, particularly the timestamped data captured in each NSSP ESSENCE visit record, to produce nowcasts of cases of a particular diagnosis of interest.
```

Comment:
Here or earlier tell me qabout the structure of this vignette. i.e. basically the subtitles

Reply:
I think this should go earlier and this statement is redundant as the section is meant to be background info